src/app/index.html

```diff
@@ -84,6 +84,7 @@
 	<script src="lib/models/generic_collection.js"></script>
 	<script src="lib/models/movie_collection.js"></script>
 	<script src="lib/models/stream_info.js"></script>
+	<script src="lib/models/webtorrent_stream_info.js"></script>
```

Comment:
stream_info.js, for instance, shows information about how many peers are and how many peers are active, but webtorrent shows number of peers (??  https://github.com/feross/webtorrent/blob/master/docs/api.md#torrentnumpeers).

I couldn't show the same information  which stream_info.js show to the user.
This is why i created another file, for comparison.
Maybe i'll need help on this file, on what information i really need to show.

Sorry if couldn't be clear enough.